lib/mix/lib/mix/tasks/format.ex

```diff
@@ -112,6 +156,71 @@ defmodule Mix.Tasks.Format do
     end
   end
 
+  # This function reads exported configuration from the imported dependencies and deals with
+  # caching the result of reading such configuration in a manifest file.
+  defp fetch_deps_opts(formatter_opts) do
+    deps = formatter_opts[:import_deps]
```

Comment:
`Keyword.get(formatter_opts, :import_deps, [])` so we can remove the nil check?

Reply:
I used this initially but removed it because I thought it made sense. Not sure anymore, so I'll just reintroduce this 😄